workflows/integration-testing/templates/restart-workflow.yaml

```diff
@@ -0,0 +1,142 @@
+        templateRef:
+          name: integration-test-job-template
+          template: inttest-template
+        arguments:
+          parameters:
+          - name: integrationtest
+            value: "-A Test_Report_Offline.list"
+          - name: jobname
+            value: cameras-offline
+      - name: standby
+        depends: cameras-offline
+        templateRef:
+          name: integration-test-job-template
+          template: inttest-template
+        arguments:
+          parameters:
+          - name: integrationtest
+            value: "-A Test_Report_Standby.list"
+          - name: jobname
+            value: standby
+      - name: disabled
+        depends: standby
+        templateRef:
+          name: integration-test-job-template
+          template: inttest-template
+        arguments:
+          parameters:
+          - name: integrationtest
+            value: "-A Test_Report_Disabled.list"
+          - name: jobname
+            value: disabled
+      - name: enabled
+        depends: disabled
+        templateRef:
+          name: integration-test-job-template
+          template: inttest-template
+        arguments:
+          parameters:
+          - name: integrationtest
+            value: "-A Test_Report_Enabled.list"
+          - name: jobname
+            value: enabled
+      - name: auxtel-housekeeping
+        depends: enabled
+        templateRef:
+          name: integration-test-job-template
+          template: inttest-template
+        arguments:
+          parameters:
+          - name: integrationtest
+            value: "-A Test_Report_AuxTel_Housekeeping.list"
+          - name: jobname
+            value: auxtel-housekeeping
+      - name: maintel-housekeeping
+        depends: enabled
+        templateRef:
+          name: integration-test-job-template
+          template: inttest-template
+        arguments:
+          parameters:
+          - name: integrationtest
+            value: "-A Test_Report_MainTel_Housekeeping.list"
+          - name: jobname
+            value: maintel-housekeeping
+      - name: auxtel-image-verification
+        depends: auxtel-housekeeping
+        templateRef:
+          name: integration-test-job-template
+          template: inttest-template
+        arguments:
+          parameters:
+          - name: integrationtest
+            value: "-A Test_Report_AuxTel_Image_Verification.list"
+          - name: jobname
+            value: auxtel-image-verification
+      - name: comcam-image-verification
+        depends: maintel-housekeeping
+        templateRef:
+          name: integration-test-job-template
+          template: inttest-template
+        arguments:
+          parameters:
+          - name: integrationtest
+            value: "-A Test_Report_ComCam_Image_Verification.list"
+          - name: jobname
+            value: comcam-image-verification
+      - name: love-stress-test
+        depends: comcam-calibrations && enable-atcs
```

Comment:
If we are going to run the LOVE stress tests on startup. this needs point to the image verification steps.